pkgs/misc/dxvk/setup_dxvk.sh

```diff
@@ -261,5 +261,5 @@ done
 
```

Comment:
Running the script with no arguments yields:
```
[nix-shell:~/git/nixpkgs]$ setup_dxvk.sh
/nix/store/yxi50hdrs81bkxz1wqgmgyidw57rhnrl-dxvk/bin/setup_dxvk.sh: line 45: $1: unbound variable
```
Could you fix that while we're here?

Reply:
Sure that shouldn't be too hard.

Reply:
Done, went through with shellcheck and fixed all warnings while I was at it.

Reply:
Your fix for this is so elegant! I wouldn't have thought of that, really nice!

([commit](https://github.com/NixOS/nixpkgs/pull/220824/commits/f67a9f0e27948fd2f16e5027c401fee80b1fef64))

Reply:
I can't really take credit for that unfortunately, the same construct was used elsewhere in the script. :P

Reply:
Oh well, learned something new :purple_heart: